model/src/main/kotlin/utils/PurlExtensions.kt

```diff
@@ -101,6 +102,8 @@ fun Provenance.toPurlExtras(): PurlExtras =
             )
         }
 
+        is DirectoryProvenance -> PurlExtras()
+
```

Comment:
I'd instead change the existing `is UnknownProvenance -> PurlExtras()` line to

    !is RemoteProvenance -> PurlExtras()

because again purls only make sense for something that has been published to a remote.